share/jupyter/nbconvert/templates/base/mathjax.html.j2

```diff
@@ -1,7 +1,8 @@
 
-{%- macro mathjax(url="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML-full,Safe") -%}
+{%- macro mathjax(mathjax_tex_url, mathjax_safe_url) -%}
```

Comment:
Thanks for making this configurable. A couple of questions about these changes:

1. I'm curious why you separated out the config into two separate script tags. My understanding is that mathjax should be loaded once, and that config argument can give multiple configurations (like `config=TeX-MML-AM_CHTML-full,Safe`), but that would be different from loading mathjax two times, each time with a different config.
2. Is this macro considered part of the public api? In other words, will user templates use this macro expecting the same signature as currently exists? Maybe we should give some default argument so users can continue using `mathjax()` in their templates and have it still function. For our templates changed in this PR, it would be loading the traitlets config values still.